presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -167,7 +167,8 @@ protected TestHiveIntegrationSmokeTest(QueryRunnerSupplier queryRunnerSupplier,
     public void testSchemaOperations()
     {
         Session admin = Session.builder(getQueryRunner().getDefaultSession())
-                .setIdentity(new Identity("hive", Optional.empty(), ImmutableMap.of("hive", new SelectedRole(SelectedRole.Type.ROLE, Optional.of("admin")))))
+                .setIdentity(Identity.forUser("hive")
+                        .withRoles(ImmutableMap.of("hive", new SelectedRole(SelectedRole.Type.ROLE, Optional.of("admin")))).build())
```

Comment:
Add `.withRole(String catalog, SelectedRole role)` to the builder.

It's a very common pattern and quite verbose today.